client/app/app.js

```diff
@@ -345,6 +345,23 @@ angular.module("managerApp", [
                 nextPage: $translate.instant("common_pagination_next")
             };
 
+            ouiFieldConfiguration.translations = {
+                errors: {
+                    required: $translate.instant("common_field_error_required"),
+                    number: $translate.instant("common_field_error_number"),
+                    email: $translate.instant("common_field_error_email"),
+                    min: $translate.instant("common_field_error_min")
+                        .replace("MIN_VALUE", "{{min}}"),
```

Comment:
why replacing `MIN_VALUE` ? Why not directly set `{{min}}` in translation ?

Reply:
Because $translate replace {{}} before we inject parameters

Reply:
I try to improve it.. I have the feeling that it will hurt translators